open-feature/openfeature.dev
# feat: add NestJS and React SDKs

## This PR

- adds React SDK readme
- adds Nest SDK readme
- updates the SDK readme parser to support framework-specific SDKs

### Notes

These framework-specific SDKs don't show up on the SDK compatibility page because they rely on an underlying SDK for the core features.

### How to test

- [NestJS](https://deploy-preview-379--openfeature.netlify.app/docs/reference/technologies/server/javascript/nestjs)
- [React](https://deploy-preview-379--openfeature.netlify.app/docs/reference/technologies/client/web/react)
- [Ecosystem](https://deploy-preview-379--openfeature.netlify.app/ecosystem?instant_search%5BrefinementList%5D%5Btype%5D%5B0%5D=SDK&instant_search%5BrefinementList%5D%5Bcategory%5D%5B0%5D=Server&instant_search%5BrefinementList%5D%5Btechnology%5D%5B0%5D=JavaScript)
- Also see the "Speaks your language" section on the main landing page.
